ui/src/main/webapp/src/app/shared/upload/file-uploader-wrapper.service.ts

```diff
@@ -60,7 +60,9 @@ export class FileUploaderWrapper extends FileUploader {
          */
 
         item._onComplete(response, status, headers);
-        this.onCompleteItem(item, response, status, headers);
+        if (!item.isCancel) {
+            this.onCompleteItem(item, response, status, headers);
+        }
```

Comment:
I think this solution is reasonable.

IMHO it is nonsense to call `onCompleteItem` when upload is canceled or an error occurs, but it it the way original `ng2-file-upload` library had it so I kept it when I wrote this wrapper.

For me :+1: 

btw maybe `item._onComplete` shouldn't be called either?